tests/strategy_tests.py

```diff
@@ -193,6 +204,39 @@ def reset_tag(self, tag):
                 db.session.delete(o)
             db.session.commit()
 
+    @pytest.fixture()
+    def load_unicode_dashboard(self):
```

Comment:
Same here (move to `fixtures.py`)

Reply:
It should be moved now. Thanks for suggestion